src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="searchbar"
+      data-qa="big"
+    >
+      <input
+        class="searchbar__input searchbar__big"
+        type="text"
+        name="searchbar__bar searchbar__big"
```

Comment:
This is weird value for name property. It is primarily used in form elements to identify input fields when they are submitted to a server.